DataFormats/L1THGCal/interface/ClusterShapes.h

```diff
@@ -0,0 +1,77 @@
+#ifndef CLUSTER_SHAPES_H
+#define CLUSTER_SHAPES_H
+#include <vector>
+#include <cmath>
+
+namespace l1t{
+    //  this class is design to contain and compute 
+    //  efficiently the cluster shapes
+    //  running only once on the cluster members.
+    class ClusterShapes{
+    private: 
+        float sum_e = 0.0;
```

Comment:
please follow a convention for member data names- eg, usually in cmssw  they end in '_'